cmd/kubeadm/app/cmd/reset.go

```diff
@@ -83,20 +76,13 @@ func NewReset(skipPreFlight, removeNode bool, certsDir string) (*Reset, error) {
 	}
 
 	return &Reset{
-		removeNode: removeNode,
-		certsDir:   certsDir,
+		certsDir: certsDir,
 	}, nil
 }
 
 // Run reverts any changes made to this host by "kubeadm init" or "kubeadm join".
 func (r *Reset) Run(out io.Writer) error {
 
-	// Try to drain and remove the node from the cluster
-	err := drainAndRemoveNode(r.removeNode)
```

Comment:
would you want to mark it unschedulable or taint it instead? seems like it should do *something* to signal that no more pods should be sent